meetings/2016-11-02.md

```diff
@@ -0,0 +1,476 @@
+
+@rvagg: Dont want to object outright, but its difficult cause I cant make
+some of the times.
+
+Has this moved more discussion to GitHub?
+
+@trott: Yes, I think so.
+
+@trott: Would like to find *3* meeting times but this has been difficult.
+
+@Fishrock123: Note that Daylight Savings Time changes are coming up in some
+place. So update the sheet accordingly.
+
+@joshgav: So is our consensus that we will continue this from now on
+indefinitely?
+
+@trott: I was going to review the times again and propose a new schedule.
+
+Everybody update the doc by this week so I can review and propose schedule.
+
+**Next steps**:
+
+* CTC members update their times in the spreadsheet by Friday 11/4.
+* @trott will choose times this weekend and share early next week.
+
+---
+
+### Review how `ctc-review` process is working [#20](https://github.com/nodejs/CTC/issues/20)
+
+@trott: This is my item and Id like to push this discussion out another week.
+
+Weve been better at doing things in GitHub, but havent had much usage of the
+CTC review process.
+
+Id like to give it another week. If we continue to not use it thats probably a
+sign we dont need it.
+
+@Fishrock123: I havent seen this used yet.
+
+@trott: That seems to support my conclusion that we cant conclude yet.
+
+**Next steps**
+
+* Discuss again in a week or two.
+
+---
+
+### lib: make `String(global) === '[object global]'` [#9279](https://github.com/nodejs/node/pull/9279)
+
+@addaleax: Change in V8 made it report [object Object]`.
+
+@Fishrock123: I dont think reverting this is necessarily a good thing, but
+perhaps we have to.
+
+OTOH it was V8 that caused the change, so maybe they should handle it.
+
+But in any case we should use the new output ([object Object]) in the next major
+version of Node.
+
+@thealphanerd: Arent people using this for environment checking?
+
+@Fishrock123: Yeah, but thats not a good way to do that.
+
+@thealphanerd: My gut says we should revert it back and prepare people for the
+change in the next major.
+
+OTOH this was caused by V8 and we havent taken responsibility for this in the
+past, would be a new precedent.
+
+@ofrobots: Two things: What to do in v7? What is the right output?
+
+For v7, we did break the ecosystem, so we should get out a fix sooner. But the
+long-term approach we can discuss.
+
+@trevnorris: Why did it return [object global], I cant replicate that anywhere
+else. Did V8 do something to cause that?
+
+@ofrobots: My understanding is that the spec required a change. I dont know the
+detailed reason.
+
+@addaleax: I think Ben dug up the source of this change, Ill see if I can find
+it.
+
+@trott: Judging from the tracker, Brian is the strongest voice opposed. Is there
+a way we can get to consensus here?
+
+@trevnorris: @bmeck do you know what part of the spec caused this change?
+
+Are we painting ourselves in the corner, since the spec change means its going
+to have to change at some point?
+
+@trott: Principal of dont break stuff on purpose.
```

Comment:
Although, honestly, not sure what I said there, but it doesn't look like it was adding much to the conversation. The line can be removed.